js/packages/seed-vault/src/useSeedVault.ts

```diff
@@ -73,9 +74,7 @@ export function useSeedVault(
         const listener = seedVaultEventEmitter.addListener(SEED_VAULT_EVENT_BRIDGE_NAME, (nativeEvent) => {
```

Comment:
Do we still need  this `useSeedVault`  + handler after the async/await API method additions?

Reply:
we still need it for event subscriptions, tho i think it makes sense to break it up and separate the content change subscription form the events. regardless, this pr is only about the async refactor.